sigh/lib/sigh/options.rb

```diff
@@ -101,7 +101,17 @@ def self.available_options
                                      env_name: "SIGH_SKIP_CERTIFICATE_VERIFICATION",
                                      description: "Skips the verification of the certificates for every existing profiles. This will make sure the provisioning profile can be used on the local machine",
                                      is_string: false,
-                                     default_value: false)
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :platform,
+                                     short_option: '-p',
+                                     env_name: "SIGH_PLATFORM",
+                                     description: "Set the provisioning profile's platform (i.e. iOS, tvOS)",
+                                     is_string: true,
+                                     default_value: "ios",
+                                     verify_block: proc do |value|
+                                       pt = %w(macos tvos ios)
```

Comment:
Does it mean that by just adding 

```ruby
value = value.to_s
```

in the `verify_block` it gets automatically passed as string when it is used by `Sigh.config[:platform]` ?